Gems/EMotionFX/Code/EMotionFX/Source/Actor.h

```diff
@@ -666,7 +666,7 @@ namespace EMotionFX
         /**
          * Get the inverse bind pose (in world space) transform of a given joint.
          * @param jointIndex The joint number, which must be in range of [0..GetNumNodes()-1].
-         * @result The inverse of the bind pose transform.
+                  * @result The inverse of the bind pose transform.
```

Comment:
Extra tab spacing change, I think this can be removed from the PR

Reply:
@cgalvan This has been resolved, can you remove your change request?